dev/install/css/install.css

```diff
@@ -1,3 +1,706 @@
ont-size:12px;padding:3px 6px;margin-left:5px;text-shadow:none;background-color:#fafbfb;-moz-border-radius-topleft:2px;-moz-border-radius-topright:2px;-moz-border-radius-bottomright:2px;-moz-border-radius-bottomleft:2px;border-radius:2px 2px 2px 2px;-webkit-box-shadow:0 0 2px 0 #7F9198;box-shadow:0 0 2px 0 #7F9198}body h5.requirement.good span{color:#359318}body h5.requirement.warning span{color:#E07127}body h5.requirement.error span{color:#C61414}body h5.requirement a{font-size:11px;right:9px;position:absolute;line-height:19px;margin:0 0 0 30px;color:#fff;text-decoration:none;font-weight:normal;padding-right:21px;background:transparent url(../../../admin/images/arrows.png) no-repeat right top}body h5.requirement a:hover{background:transparent url(../../../admin/images/arrows.png) no-repeat right -40px}body table.testResults{border-top:none;margin-top:-1px;margin-bottom:9px;-webkit-box-shadow:0 1px 0 0 #fff;box-shadow:0 1px 0 0 #fff;-moz-border-radius-topleft:3px;-moz-border-radius-topright:3px;-moz-border-radius-bottomright:3px;-moz-border-radius-bottomleft:3px;border-radius:3px 3px 3px 3px;overflow:hidden}body table.testResults tr.good{display:none;border:none}body table.testResults tr.good td{color:#359318}body table.testResults tr.warning{border:none}body table.testResults tr.warning td{color:#E07127}body table.testResults tr.error{border:none;color:#C61414}body table.testResults td{border:1px solid #ddd;width:50%}body #database_selection,body #Themes{list-style:none;margin:0;margin-bottom:18px;padding-left:0}body #database_selection li,body #Themes li{clear:left;padding:3px 0}body #database_selection li .dbfields,body #Themes li .dbfields{padding:12px;border:1px solid #ddd;background-color:#fafafa;margin-bottom:9px;-moz-border-radius-topleft:3px;-moz-border-radius-topright:3px;-moz-border-radius-bottomright:3px;-moz-border-radius-bottomleft:3px;border-radius:3px 3px 3px 3px;-webkit-box-shadow:inset 0 0 3px 0 #ddd;box-shadow:inset 0 0 3px 0 #ddd;-webkit-box-shadow:0 1px 0 0 #fff;box-shadow:0 1px 0 0 #fff}body ul#Themes{float:left;width:445px}body .databaseError{width:422px}body #Footer{margin-top:67px;margin-bottom:18px}body #Footer p{font-size:12px;color:#999}body #Header h1{margin-top:45px;margin-bottom:27px}body #Header h3{margin-top:-10px}#adminAcc,#devHelp,#localeHelp{padding-top:20px}#devHelp,#devSection{height:18em}#use_environment_field{margin-bottom:25px;background-color:#FAFAFA;border:1px solid #DDDDDD;border-radius:3px 3px 3px 3px;-webkit-box-shadow:0 1px 0 0 white;box-shadow:0 1px 0 0 white;margin-bottom:9px;padding:10px 10px 8px}#use_environment_field input{float:left;margin-right:5px}.databaseError ul{margin-bottom:0}.databaseError ul,.databaseError li{margin-left:0;padding-left:0}#Layout p.goodInstall{color:#359318}.goodInstall{padding:0.5em;background-color:#e2fee1;border:1px #43cb3e solid;color:#359318;overflow:hidden;line-height:18px;padding:10px}.goodInstall a{float:right;font-size:18px;padding:0 2px 2px 0;font-weight:bold}
```

Comment:
ok done, didn't realise --development doesn't minify the css. I need to check my other pull requests.